GrafCalculator/app/src/main/java/dev/software/bing/CalculatorFragment.kt

```diff
@@ -0,0 +1,122 @@
+package dev.software.bing
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import dev.software.bing.databinding.FragmentCalculatorBinding
+
+
+class CalculatorFragment : Fragment() {
+
+    var resultList: ArrayList<String> = arrayListOf()
+
+    private var _binding: FragmentCalculatorBinding? = null
+    private val binding
+        get() = requireNotNull(_binding)
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return FragmentCalculatorBinding.inflate(inflater, container, false)
+            .also { _binding = it  }
+            .root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        with(binding){
+            btnClear.setOnClickListener{
+                inputTextView.text = ""
+            }
+            btnLeftBracket.setOnClickListener{
+                inputTextView.text = inputTextView.text.toString() +btnLeftBracket.text
```

Comment:
лучше использовать функцию append, ниже будет аналогично
```suggestion
                inputTextView.append(btnLeftBracket.text)
```